.travis.yml

```diff
@@ -43,7 +43,7 @@ script:
       bash .misc/tests.sh
       python setup.py bdist_wheel
       pip install ./dist/coala-*.whl
-      pip install coala-bears --pre -U
+      pip install coala-bears[alldeps] --pre
```

Comment:
putting a version here makes no difference at all.  `coala-bears` isnt installed, so the `-U` has no effect wrt that.

 `-U` upgrades the deps to newer versions.  Not using `-U` will mean we get a `VersionConflict` if pip should have installed a newer version than was already installed, indicating that our requirements between the two packages are slightly problematic.   Problems like https://github.com/coala/coala/pull/3444 will be found sooner (if it happens in real requirements, not test-requirements).

We really need https://github.com/coala/coala-bears/issues/1113 , or that can be built into cib.

(and we should enable PySafetyBear)